shell/java-hello-world/java-hello-world.task.yaml

```
@@ -0,0 +1,9 @@
+name: "[Shell] Java hello world"
+slug: java_hello_world
+parameters:
+  - name: Name
+    slug: name
+    type: shorttext
```

```suggestion
slug: java_hello_world
description: Greets you by name!
parameters:
  - name: Name
    slug: name
    type: shorttext
    default: World
```
I've also been using `<slug>.task.yaml` for these, so maybe underscores instead?